src/propagation/serialization.jl

```diff
@@ -0,0 +1,204 @@
+@doc raw"""
+    TaylorInterpolantNSerialization{T}
+
+Custom serialization struct to save a `TaylorInterpolant{T, TaylorN{T}, 2}` to a `.jld2` file. 
+
+# Fields
+- `vars::Vector{String}`: jet transport variables. 
+- `order::Int`: order of Taylor polynomials.
```

Comment:
I'd add here "w.r.t. time" at the end of the phrase

Reply:
Done.